lib/atomic_cache/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module AtomicCache
-  VERSION = "0.1.0.rc2"
+  VERSION = "0.1.0.rc1"
```

Comment:
changing this back just to make sure the `pull_request` conditional build works before I flip it back.